package.json

```diff
@@ -73,22 +73,23 @@
   ],
   "devDependencies": {
     "@axe-core/playwright": "^4.4.4",
-    "@babel/core": "^7.16.0",
-    "@babel/plugin-proposal-class-properties": "^7.12.1",
-    "@babel/plugin-transform-object-assign": "^7.16.0",
-    "@babel/preset-env": "^7.8.4",
-    "@babel/preset-react": "^7.8.3",
-    "@babel/preset-typescript": "^7.15.0",
+    "@babel/core": "^7.21.4",
+    "@babel/helper-string-parser": "^7.19.4",
+    "@babel/plugin-proposal-decorators": "^7.21.0",
+    "@babel/plugin-proposal-export-default-from": "^7.18.10",
+    "@babel/plugin-syntax-flow": "^7.21.4",
+    "@babel/preset-env": "^7.21.4",
+    "@babel/preset-react": "^7.18.6",
+    "@babel/preset-typescript": "^7.21.4",
     "@babel/register": "^7.8.4",
-    "@playwright/test": "^1.31.0",
-    "@storybook/addon-actions": "^6.5.16",
-    "@storybook/addon-essentials": "^6.5.16",
-    "@storybook/addon-links": "^6.5.16",
-    "@storybook/builder-webpack5": "^6.5.16",
-    "@storybook/manager-webpack5": "^6.5.16",
-    "@storybook/preset-scss": "^1.0.3",
-    "@storybook/react": "^6.5.16",
-    "@storybook/theming": "^6.5.16",
+    "@playwright/test": "^1.33.0",
```

Comment:
SB migration bumped this package and it's breaking tests in CI 😩 Addressing this issue now